app/services/idv/send_phone_confirmation_otp.rb

```diff
@@ -0,0 +1,96 @@
+module Idv
+  # Ignore instance variable assumption on @user_locked_out :reek:InstanceVariableAssumption
+  class SendPhoneConfirmationOtp
+    attr_accessor :user, :idv_session, :locale
```

Comment:
Since these methods are not meant to be writable outside of the class, what do you think about replacing this with a private `attr_reader`, and using `@user = user` etc. in the `initialize` method?